tests/e2e/utils/pages/setup-mc/step-4-complete-campaign.js

```diff
@@ -112,17 +88,6 @@ export default class CompleteCampaign extends MockRequests {
 		} );
 	}
 
-	/**
-	 * Click skip this step for now button.
-	 *
-	 * @return {Promise<void>}
-	 */
-	async clickSkipStepButton() {
```

Comment:
Keep this method once the `getSkipStepButton` is updated above and the test for skipping is reinstated.

Reply:
Same @joemcgill , we already have a function for the skip button:
https://github.com/woocommerce/google-listings-and-ads/blob/977d9e16df5142a0c174d75b71e3fb183918dad2/tests/e2e/utils/pages/setup-mc/step-4-complete-campaign.js#L91-L100